debian/changelog

```diff
@@ -1,5 +1,6 @@
-cloud-init (21.1-19-gbad84ad4-0ubuntu1) UNRELEASED; urgency=medium
-
+cloud-init (21.1-19-gbad84ad4-0ubuntu1) hirsute; urgency=medium
+  * d/cloud-init.postinst: debian/cloud-init.postinst: Change output log
```

Comment:
I appreciate the thorough approach, but we can probably just mention the file path once. :grin: